foundation/foundation-mda/src/main/resources/templates/deployment/argocd/spark-operator.yaml.vm

```diff
@@ -23,5 +23,21 @@ spec:
     {{ end }}
   {{- with .Values.spec.syncPolicy }}
   syncPolicy:
-    {{- toYaml . | nindent 4 }}
-  {{- end }}
\ No newline at end of file
+    {{- with .Values.spec.syncPolicy }}
+      {{- if .automated }}
+    automated:
+        {{- toYaml .automated | nindent 6}}
+      {{- else if hasKey . "automated" }}
+    automated: {}
+      {{- end }}
+    {{- end }}
+    {{/* This template comment explains how the logic below works*/}}
+    {{- /* This one trims the newlines so it doesn't affect output */ -}}
+    #these options are important
+    syncOptions:
+      - ServerSideApply=true # https://github.com/argoproj/argo-cd/issues/820#issuecomment-135463693
+    {{- if .Values.spec.syncPolicy }}
+      {{- with .Values.spec.syncPolicy.syncOptions }}
+        {{- toYaml . | nindent 6 }}
+      {{- end }}
+    {{- end }}
```

Comment:
I: this is a missed change when migrating the syncOptions for the spark-operator

Reply:
See the _Known Issues_ for [release 1.8](https://github.com/boozallen/aissemble/releases/tag/aissemble-root-1.8.0).  This may need to be backported to 1.9.